js/main.js

```diff
@@ -22,4 +22,67 @@ $form.addEventListener('submit', function (event) {
   data.entries.unshift(newObject);
   $imgSrc.setAttribute('src', 'images/placeholder-image-square.jpg');
   $form.reset();
+  const $ul = document.querySelector('ul');
```

Comment:
This should be outside the function in the global space. By including it here, you're traversing the DOM over and over every time this function gets called. If you do it in the global space, you only do the search once and we always have a reference to it.